test/autoCompletion.test.ts

```diff
@@ -558,8 +560,8 @@ describe('Auto Completion Tests', () => {
             },
           },
         });
-        const content = 'scripts: |:|';
-        const completion = parseSetup(content);
+        const content = 'scripts: ';
+        const completion = parseSetup(content, content.length);
```

Comment:
I have no way of being true to the original test here, since `content[content.length]` is (intentionally it would seem) out of bounds.

I can either leave it as-is, or I can perhaps change the content to:
`scripts: |:|` which I believe would be the moral equivalent?